frappe/model/db_query.py

```diff
@@ -684,21 +680,12 @@ def apply_fieldlevel_read_permissions(self):
 			elif "(" in field:
 				if "*" in field:
 					continue
-				elif _params := FN_PARAMS_PATTERN.findall(field):
-					params = (x.strip() for x in _params[0].split(","))
-					for param in params:
-						if not (
-							not param
-							or param in permitted_fields
-							or param.isnumeric()
-							or "'" in param
-							or '"' in param
-						):
+				else:
+					for column in columns:
+						if not column in permitted_fields:
```

Comment:
`tabPhoto.name` seems to already work because we split it if we see `.` in fieldname. 

https://github.com/ankush/frappe/blob/ebd47429a54ac602ae247198319c31acd137b1c9/frappe/model/db_query.py#L660-L661


This branch is mostly unused now. Will keep it for now though.